docs/pages/x/api/date-pickers/pickers-calendar-header.json

```diff
@@ -0,0 +1,28 @@
+{
+  "props": {
+    "classes": { "type": { "name": "object" }, "additionalPropsInfo": { "classes": true } },
+    "className": { "type": { "name": "string" } },
+    "slotProps": { "type": { "name": "object" }, "default": "{}" },
+    "slots": { "type": { "name": "object" }, "default": "{}" },
+    "sx": {
+      "type": {
+        "name": "union",
+        "description": "Array&lt;func<br>&#124;&nbsp;object<br>&#124;&nbsp;bool&gt;<br>&#124;&nbsp;func<br>&#124;&nbsp;object"
+      },
+      "additionalPropsInfo": { "sx": true }
+    }
+  },
+  "slots": [],
```

Comment:
Isn't it missing the available slots declaration? 🤔

Reply:
The doc generation is still using `components` and `componentsProps`
It's probably worth creating an issue to migrate to `slots` / `slotProps` :+1: